sw/device/lib/otbn.h

```diff
@@ -0,0 +1,248 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_otbn.h"
+
+/**
+ * @file
+ * @brief OpenTitan Big Number Accelerator (OTBN) driver
+ */
+
+/**
+ * Information about an embedded OTBN application image.
+ *
+ * All pointers reference data in the normal CPU address space.
+ *
+ * Use `OTBN_USE_APP()` together with `OTBN_APP_T_INIT()` to initialize this
+ * structure.
+ */
+typedef struct otbn_app {
+  /**
+   * Start of OTBN instruction memory.
+   */
+  const char *imem_start;
```

Comment:
We assume that uint8_t is a character type, so perhaps use that, instead?

Reply:
The symbol that gets assigned here is char as well, so using char keeps things consistent.

Reply:
Changed char to `uint8_t`.